nifi-nar-bundles/nifi-azure-bundle/nifi-azure-services-api/src/main/java/org/apache/nifi/services/azure/storage/ADLSCredentialsDetails.java

```diff
@@ -98,6 +100,39 @@ public String getServicePrincipalClientSecret() {
         return servicePrincipalClientSecret;
     }
 
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
```

Comment:
Please use the regular formatting and always add `{}` in the `if` statement even if it is an IDE generated code.